### ISSUE TITLE
Bodymovin stuck on rendering Auto-traced layer

### ISSUE THREAD

**Issue by fordspencer** [airbnb/lottie-web#2229]
I am trying to render a simple path animation that was auto traced. It also has a 4 color gradient effect on it. But I tried rendering it out without that effect and it still gets stuck and I have to force quit After Effects to get out of it. I assume it has something to do with the auto-trace masks or keyframes? I went over the list of things that are not supported and didnt find any problems. Any ideas?

The animation is just two layers, first one is the auto traced path which is composed to 2 masks set to add, and the second layer is a solid bg with no animation. Total composition time is 3 seconds. 

<img width="404" alt="Screenshot 2020-07-13 at 13 32 58" src="https://user-images.githubusercontent.com/68225513/87300892-f2af2500-c50e-11ea-9287-d96ea81a24b6.png">
<img width="836" alt="Screenshot 2020-07-13 at 13 42 13" src="https://user-images.githubusercontent.com/68225513/87300921-ff337d80-c50e-11ea-93f0-8cc02b613698.png">
<img width="264" alt="Screenshot 2020-07-13 at 13 35 53" src="https://user-images.githubusercontent.com/68225513/87300931-022e6e00-c50f-11ea-97d3-c80db3488e06.png">

**Comment by bodymovin** [comment]
hi, can you share the .aep?
I'll check it as soon as I can

**Comment by fordspencer** [comment]
[blob-gradient-01 folder.zip](https://github.com/airbnb/lottie-web/files/4912356/blob-gradient-01.folder.zip)

**Comment by bodymovin** [comment]
it looks like you shared an empty project

**Comment by fordspencer** [comment]
Ah damn. Well I think I figured it out In the meantime. The layer has a lot of moving blobs that have all been auto traced, so each shape has many many vertices as well as a keyframe per frame because of the auto-trace function. So I think its just way too much information and the Bodymovin doesn't crash but just takes ages to render it out and when it does the files are huge, to the point where it makes no sense to do it this way. I have found that auto trace only works well with bodymovin in very small doses. So I have changed the animation style to not include auto-trace and to work better with Lottie and am moving forward that way. Thanks anyways!
